bindings/wasm/src/lib.rs

```diff
@@ -87,7 +87,7 @@ impl Client {
 
     #[wasm_bindgen(js_name = "getNodeInfo")]
     pub async fn get_node_info(self) -> Result<JsValue, JsValue> {
-        let node_info = self.client.get_node_info()
+        let node_info = iota::client::client::Client::get_node_info()
```

Comment:
I think `iota-core` already re-export the `Client`. You don't need to add the path that long.
```suggestion
        let node_info = iota::Client::get_node_info()
```